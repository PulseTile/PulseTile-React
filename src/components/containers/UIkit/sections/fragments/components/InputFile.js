import React from 'react';

/**
 * This component returns content of InputFile section
 */
const InputFile = () => {
  return (
    <div id="input-file" className="ui-section">
      <strong className="ui-title">Input File</strong>
      <div className="form-group-wrapper">
        <div className="row">
          <div className="col-xs-12 col-sm-6">
            <div className="form-group">
              <label className="control-label">Application File</label>
              <div className="input-holder">
                <div className="wrap-fcustomfile">
                  <div className="fcustomfile-control">
                    <label for="logoPath" className="btn btn-success btn-inverse btn-normal-icon">
                      <input accept="image/jpeg,image/png,image/gif" type="file" name="logoPath" id="logoPath" />
                      <i className="fa fa-plus"></i> <span>Upload file</span>
                    </label>
                  </div>
                  <div className="fcustomfile-text"></div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default InputFile;