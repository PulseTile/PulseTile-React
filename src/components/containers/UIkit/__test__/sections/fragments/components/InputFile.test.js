import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import InputFile from '../../../../sections/fragments/components/InputFile';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <InputFile />', () => {
  it('should renders InputFile with props correctly', () => {
    const component = shallow(<InputFile />);
    expect(component).toMatchSnapshot();
  });
});
