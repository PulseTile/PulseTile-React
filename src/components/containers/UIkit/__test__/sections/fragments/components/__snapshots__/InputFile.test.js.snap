// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <InputFile /> should renders InputFile with props correctly 1`] = `
<div
  className="ui-section"
  id="input-file"
>
  <strong
    className="ui-title"
  >
    Input File
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
          >
            Application File
          </label>
          <div
            className="input-holder"
          >
            <div
              className="wrap-fcustomfile"
            >
              <div
                className="fcustomfile-control"
              >
                <label
                  className="btn btn-success btn-inverse btn-normal-icon"
                  for="logoPath"
                >
                  <input
                    accept="image/jpeg,image/png,image/gif"
                    id="logoPath"
                    name="logoPath"
                    type="file"
                  />
                  <i
                    className="fa fa-plus"
                  />
                   
                  <span>
                    Upload file
                  </span>
                </label>
              </div>
              <div
                className="fcustomfile-text"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
